samples/android/Resources/xml/preferences.axml

```diff
@@ -1,93 +1,56 @@
 <?xml version="1.0" encoding="utf-8"?>
-<PreferenceScreen xmlns:android="http://schemas.android.com/apk/res/android" >
```

Comment:
Again, these changes look bigger than they actually are. The structure is the same, just adapted to the v7 preference syntax.